pkg/flux/flux.go

```diff
@@ -268,20 +224,6 @@ func (f *FluxClient) runFluxCmd(args ...string) ([]byte, error) {
 	return out, nil
 }
 
-func (f *FluxClient) runFluxCmdOutputStream(args ...string) ([]byte, error) {
-	fluxPath, err := f.fluxPath()
-	if err != nil {
-		return []byte{}, errors.Wrap(err, "error getting flux binary path")
-	}
-
-	out, err := f.runner.RunWithOutputStream(fluxPath, args...)
-	if err != nil {
-		return []byte{}, fmt.Errorf("failed to run flux with output: %s and error: %w", string(out), err)
-	}
-
-	return out, nil
-}
-
 func (f *FluxClient) fluxPath() (string, error) {
```

Comment:
fluxPath can be removed as well. Since we dont have the binary embedded anymore. Whether its part of this or not is fine with me.

Reply:
The embedded binary is still in there - I've avoided doing anything to it since we talked on Friday about the need for _some_ "verified flux", and I don't think we decided what that is. So gonna leave it right now.